.idea/misc.xml

```diff
@@ -6,9 +6,9 @@
       <list>
         <option value="$PROJECT_DIR$/pom.xml" />
         <option value="$PROJECT_DIR$/kroxylicious-systemtests/pom.xml" />
+        <option value="$PROJECT_DIR$/performance-tests/dependency-reduced-pom.xml" />
```

Comment:
I suspect your tree  has some stale files.  `performance-tests` is no longer a Maven module so there shouldn't be a dependency-reduced-pom.xml inside it..